include/unifex/v1/async_scope.hpp

```diff
@@ -0,0 +1,609 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License Version 2.0 with LLVM Exceptions
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *   https://llvm.org/LICENSE.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <unifex/config.hpp>
+#include <unifex/async_manual_reset_event.hpp>
+#include <unifex/get_stop_token.hpp>
+#include <unifex/inplace_stop_token.hpp>
+#include <unifex/just_from.hpp>
+#include <unifex/let_value_with.hpp>
+#include <unifex/let_value_with_stop_token.hpp>
+#include <unifex/manual_lifetime.hpp>
+#include <unifex/nest.hpp>
+#include <unifex/on.hpp>
+#include <unifex/receiver_concepts.hpp>
+#include <unifex/scheduler_concepts.hpp>
+#include <unifex/sender_concepts.hpp>
+#include <unifex/sequence.hpp>
+#include <unifex/spawn_detached.hpp>
+#include <unifex/spawn_future.hpp>
+#include <unifex/then.hpp>
+#include <unifex/type_traits.hpp>
+
+#include <atomic>
+#include <memory>
+#include <tuple>
+
+#include <unifex/detail/prologue.hpp>
+
+namespace unifex {
+
+inline namespace v1 {
```

Comment:
In `main`, the `v0` namespace is `inline`, right?  If so, I think this has to be non-inline.
```suggestion
namespace v1 {
```